### DISCUSSION TITLE
How to specify location for the test here using JMeter DSL with .NET?

### DISCUSSION THREAD

**Discussion by anoncheva** [abstracta/jmeter-dotnet-dsl#26] [Q&A] [answered]
I see Locations do not seem added at all here in BM engine implementation for .NET
While they are there for the Java version: https://github.com/abstracta/jmeter-java-dsl/blob/master/jmeter-java-dsl-blazemeter/src/main/java/us/abstracta/jmeter/javadsl/blazemeter/BlazeMeterEngine.java 
So do you plan to implement Locations for .NET as well?

**Comment by rabelenda** [comment, marked as answer]
Hello,

We just released a new version of the DSL that allows yo set the location as in the Java DSL.

Regards
